pkg/controllers/netpol/nftables.go

```diff
@@ -372,6 +384,20 @@ func getOrCreateSet(unfiltered []string) string {
 	return setName
 }
 
+func fixOverlappingNets(nets *[]*net.IPNet, network *net.IPNet) {
```

Comment:
I think I need a prose explanation of this function to understand what it's fixing and why it's okay.